libraries/SITL/SITL.cpp

```diff
@@ -472,28 +477,82 @@ const AP_Param::GroupInfo SIM::var_gps[] = {
     // @Param: GPS_BYTELOSS
     // @DisplayName: GPS Byteloss
     // @Description: Percent of bytes lost from GPS 1
+    // @Units: %
     // @User: Advanced
     AP_GROUPINFO("GPS_BYTELOSS",   4, SIM,  gps_byteloss[0],  0),
     // @Param: GPS_NUMSATS
     // @DisplayName: GPS 1 Num Satellites
     // @Description: Number of satellites GPS 1 has in view
     AP_GROUPINFO("GPS_NUMSATS",    5, SIM,  gps_numsats[0],   10),
+    // @Param: GPS_GLITCH
+    // @DisplayName: GPS 1 Glitch
+    // @Description: Glitch offsets of simulated GPS 2 sensor
```

Comment:
Txs for this.  I guess it's "GPS 1 sensor"?

Reply:
oops - I messed that one up. Should be fixed now!